herd/machAction.ml

```diff
@@ -114,6 +115,12 @@ end = struct
           if kvm then Access.PTE
           else Warn.fatal "PTE %s while -variant kvm is not active"
                  (A.pp_location loc)
+    | A.Location_global (V.Val (Label(_,_))) as loc
+        ->
+          Warn.warn_always
+            "FIXME: access_of_location_std on a label '%s' to be verified with -variant self"
+            (A.pp_location loc)
+          ; VIR
```

Comment:
The function is useful for kvm mode (model of virtual memory), You can get rid of the warning (which doesnot occur for writes anyway, as far as I have checked).

Reply:
Coming in the next commit.